auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/response/PaginatedResponse.java

```diff
@@ -0,0 +1,49 @@
+package com.atlantbh.auctionappbackend.response;
+
+import java.util.List;
+
+public class PaginatedResponse<T> {
+    private List<T> products;
```

Comment:
Since we are not going to use this class only for products, rename this to something more generic (e.g. `data`).